### PR TITLE
updates to GSoC Page after selection into the program

### PR DESCRIPTION
**Pull request by rareddy**
This updates page after the selection Kubeflow into GSoc mainly highlighting

- how to participate in kubeflow's gsoc
- how to reach out to mentors
- how to reach out to mentors in community calls
- what to do if student did not get selected in gsoc program

### CODE REVIEW

**Comment by rareddy** [comment]
@thesuperzapper can you review this pls

File: content/en/events/gsoc-2024.md [hunk 1]
```diff
@@ -5,25 +5,46 @@ description = "Google Summer of Code 2024"
...
 Below are the "ideas/features" from different projects in the Kubeflow that being submitted for seeking proposals from GSoC Students.
 
-## Registration
+## How can I participate in Kubeflow GSoc?
 
-Please go to [Google Summer of Code 2024](https://buildyourfuture.withgoogle.com/programs/summer-of-code) and sign up as student and look up Kubeflow organization and select the projects you want to submit proposals. Note, you can submit more than one proposal. Contributor Applications open **March 20, 2024** and end on **Apr 4th 2024**. So start writing your proposals.
+Please go to [Google Summer of Code 2024](https://buildyourfuture.withgoogle.com/programs/summer-of-code) and sign up as student and look up Kubeflow organization and select the projects you want to submit proposals. Note, you can submit more than one proposal. Contributor Applications open **March 18th, 2024** and end on **Apr 2nd 2024**. First read through contributor guidelines, how to participate GSoC google website. Only ask questions about projects with your mentors. General questions about GSoC please reach out to GSoC organizers.
```
Comment (thesuperzapper) [inline content/en/events/gsoc-2024.md:15]: Reworded:

```suggestion
Please go to [Google Summer of Code 2024](https://buildyourfuture.withgoogle.com/programs/summer-of-code) and sign up as a student. Next, look at the [projects below](#project-ideas-for-2024-gsoc) to decide which ones you are interested in. Note, that you must submit your proposals through the GSoC website and your proposal must be selected to participate.

Contributor applications open __March 18th, 2024__ and close on __April 2nd, 2024__. 

For more information, see the GSoC website and/or reach out to the GSoC organizers.
Please only contact mentors about projects, not the program itself.
```

File: content/en/events/gsoc-2024.md [hunk 2]
```diff
@@ -5,25 +5,46 @@ description = "Google Summer of Code 2024"
...
 
-## Students
+## Meetings
 
-Welcome to an incredible chance to dive into the vibrant Kubeflow community! Join us and become a valued contributor to our rapidly expanding network dedicated to AI/ML technologies on Kubernetes. Kubeflow mentors come with years of valuable experience and a passion for community growth and project enhancement, they're the guiding stars of our program. Immerse yourself in a dynamic environment where you'll not only gain insights from top industry experts but also sharpen your skills in open source collaboration, time management, and presentation prowess alongside making impactful technical contributions to projects of your interest. Our dedicated mentors will be by your side every step of the way, offering invaluable guidance and support throughout the project. Get ready to embark on an exciting journey of growth and learning!
+To facilitate the questions from students, we ask you to find the next community meeting for the project you are interested in from below list and attend that meeting and ask questions.
```
Comment (thesuperzapper) [inline content/en/events/gsoc-2024.md:23]: Reworded:

```suggestion
You may wish to attend the next community meeting for the group that is leading your chosen project, please see the calendar below for more information.
```

File: content/en/events/gsoc-2024.md [hunk 3]
```diff
@@ -5,25 +5,46 @@ description = "Google Summer of Code 2024"
...
+  // KServe Community
+  "4fqdmu5fp4l0bgdlf4lm1atnsl2j4612%40import.calendar.google.com",
+];
+let calender_src = calender_src_list.map(src => `&src=${src}&color=%23A79B8E`).join('');
+const html = `<iframe src="https://calendar.google.com/calendar/embed?ctz=${timezone}&height=600&wkst=1&bgcolor=%23ffffff&showPrint=0&showDate=1&mode=AGENDA&showTitle=0${calender_src}" style="border:solid 1px #777" width="800" height="600" frameborder="0" scrolling="no"></iframe>`;
+document.getElementById('calendar-container').innerHTML = html;
+</script>
 
-## Project Ideas for the 2024
+## Didn't get selected for GSoC? Don't worry, you can still contribute!
```
Comment (thesuperzapper) [inline content/en/events/gsoc-2024.md:39]: This looks glitchy on the website, lets make it shorter:

```suggestion
## What if my proposal is not chosen?
```

File: content/en/events/gsoc-2024.md [hunk 4]
```diff
@@ -5,25 +5,46 @@ description = "Google Summer of Code 2024"
...
+
+We understand that not everyone gets selected for Google Summer of Code (GSoC). However, we're still open to collaboration with enthusiastic students! While we can't offer a stipend like GSoC, we can provide valuable mentorship and experience.
```
Comment (thesuperzapper) [inline content/en/events/gsoc-2024.md:41]:
```suggestion
Please understand that not everyone can be selected for Google Summer of Code (GSoC), there are many possible candidates for each project.
However, we want to encourage you to participate in the Kubeflow project!
The best way to get started is to attend [working group meetings](/docs/about/community/#kubeflow-community-calendars) for components you want to contribute to.
```

File: content/en/events/gsoc-2024.md [hunk 5]
```diff
@@ -5,25 +5,46 @@ description = "Google Summer of Code 2024"
...
+
+To ensure successful mentoring, we expect a strong commitment and enthusiasm from you for the chosen project. This way, our mentors can dedicate their time and expertise to guide you effectively.
+
+We believe there are many ways to contribute outside of GSoC. If you're interested, please reach out to us!
+
+# Project Ideas for the 2024
```
Comment (thesuperzapper) [inline content/en/events/gsoc-2024.md:47]: This was incorrect English: 

```suggestion
# Project Ideas for 2024 GSoC
```

File: content/en/events/gsoc-2024.md [hunk 6]
```diff
@@ -5,25 +5,46 @@ description = "Google Summer of Code 2024"
...
+</script>
 
-## Project Ideas for the 2024
+## Didn't get selected for GSoC? Don't worry, you can still contribute!
+
+We understand that not everyone gets selected for Google Summer of Code (GSoC). However, we're still open to collaboration with enthusiastic students! While we can't offer a stipend like GSoC, we can provide valuable mentorship and experience.
+
+To ensure successful mentoring, we expect a strong commitment and enthusiasm from you for the chosen project. This way, our mentors can dedicate their time and expertise to guide you effectively.
+
+We believe there are many ways to contribute outside of GSoC. If you're interested, please reach out to us!
```
Comment (thesuperzapper) [inline content/en/events/gsoc-2024.md:45]: This is not really related to the section, let's remove it:

```suggestion
```

File: content/en/events/gsoc-2024.md [hunk 7]
```diff
@@ -5,25 +5,46 @@ description = "Google Summer of Code 2024"
...
 Below are the "ideas/features" from different projects in the Kubeflow that being submitted for seeking proposals from GSoC Students.
 
-## Registration
+## How can I participate in Kubeflow GSoc?
 
-Please go to [Google Summer of Code 2024](https://buildyourfuture.withgoogle.com/programs/summer-of-code) and sign up as student and look up Kubeflow organization and select the projects you want to submit proposals. Note, you can submit more than one proposal. Contributor Applications open **March 20, 2024** and end on **Apr 4th 2024**. So start writing your proposals.
+Please go to [Google Summer of Code 2024](https://buildyourfuture.withgoogle.com/programs/summer-of-code) and sign up as student and look up Kubeflow organization and select the projects you want to submit proposals. Note, you can submit more than one proposal. Contributor Applications open **March 18th, 2024** and end on **Apr 2nd 2024**. First read through contributor guidelines, how to participate GSoC google website. Only ask questions about projects with your mentors. General questions about GSoC please reach out to GSoC organizers.
 
-## Mentors
+## Slack
```
Comment (thesuperzapper) [inline content/en/events/gsoc-2024.md:17]: Let's make this H3, so it goes under `How can I participate in Kubeflow GSoc?`:

```suggestion
### Slack
```

File: content/en/events/gsoc-2024.md [hunk 8]
```diff
@@ -5,25 +5,46 @@ description = "Google Summer of Code 2024"
...
 
+Students, welcome to an incredible chance to dive into the vibrant Kubeflow community! Join us and become a valued contributor to our rapidly expanding network dedicated to AI/ML technologies on Kubernetes. Kubeflow mentors come with years of valuable experience and a passion for community growth and project enhancement, they're the guiding stars of our program. Immerse yourself in a dynamic environment where you'll not only gain insights from top industry experts but also sharpen your skills in open source collaboration, time management, and presentation prowess alongside making impactful technical contributions to projects of your interest. Our dedicated mentors will be by your side every step of the way, offering invaluable guidance and support throughout the project. Get ready to embark on an exciting journey of growth and learning!
 Below are the "ideas/features" from different projects in the Kubeflow that being submitted for seeking proposals from GSoC Students.
 
-## Registration
+## How can I participate in Kubeflow GSoc?
```
Comment (thesuperzapper) [inline content/en/events/gsoc-2024.md:13]: Let's style as `GSoC` (different caps):

```suggestion
## How can I participate in Kubeflow GSoC?
```

File: content/en/events/gsoc-2024.md [hunk 9]
```diff
@@ -5,25 +5,46 @@ description = "Google Summer of Code 2024"
...
 
+Students, welcome to an incredible chance to dive into the vibrant Kubeflow community! Join us and become a valued contributor to our rapidly expanding network dedicated to AI/ML technologies on Kubernetes. Kubeflow mentors come with years of valuable experience and a passion for community growth and project enhancement, they're the guiding stars of our program. Immerse yourself in a dynamic environment where you'll not only gain insights from top industry experts but also sharpen your skills in open source collaboration, time management, and presentation prowess alongside making impactful technical contributions to projects of your interest. Our dedicated mentors will be by your side every step of the way, offering invaluable guidance and support throughout the project. Get ready to embark on an exciting journey of growth and learning!
 Below are the "ideas/features" from different projects in the Kubeflow that being submitted for seeking proposals from GSoC Students.
```
Comment (thesuperzapper) [inline content/en/events/gsoc-2024.md:11]:
```suggestion
Below are the "ideas/features" from different projects in Kubeflow that are seeking proposals from GSoC Students.
```

**Review by thesuperzapper — CHANGES_REQUESTED** [review]
@rareddy made a few suggestions.

File: content/en/events/gsoc-2024.md [hunk 10]
```diff
@@ -5,25 +5,46 @@ description = "Google Summer of Code 2024"
...
+Kubeflow Community is excited to announce that we have been [selected](https://summerofcode.withgoogle.com/programs/2024/organizations/kubeflow) as organization to participate in [**Google Summer of Code 2024**](https://buildyourfuture.withgoogle.com/programs/summer-of-code).
 
+Students, welcome to an incredible chance to dive into the vibrant Kubeflow community! Join us and become a valued contributor to our rapidly expanding network dedicated to AI/ML technologies on Kubernetes. Kubeflow mentors come with years of valuable experience and a passion for community growth and project enhancement, they're the guiding stars of our program. Immerse yourself in a dynamic environment where you'll not only gain insights from top industry experts but also sharpen your skills in open source collaboration, time management, and presentation prowess alongside making impactful technical contributions to projects of your interest. Our dedicated mentors will be by your side every step of the way, offering invaluable guidance and support throughout the project. Get ready to embark on an exciting journey of growth and learning!
```
[the remark this answers is not in the document]
Comment (rareddy) [reply content/en/events/gsoc-2024.md]: removed, and also accepted the following suggestions! Thank you.

File: content/en/events/gsoc-2024.md [hunk 11]
```diff
@@ -5,25 +5,53 @@ description = "Google Summer of Code 2024"
...
+### Slack
 
-**If you need to reach out our mentors, join [Kubeflow Slack](https://www.kubeflow.org/docs/about/community/#kubeflow-slack) and look up your mentor or find project specific channel on Kubeflow slack**
+Please [__Join the Kubeflow Slack__](https://www.kubeflow.org/docs/about/community/#kubeflow-slack).
 
-## Project Ideas for the 2024
+Please __do not__ reach out privately to mentors, instead, start a thread in the `#gsoc-participants` channel so others can see the response. Be kind to our mentors, please search to see if your question has already been answered.
```
Comment (andreyvelich) [inline content/en/events/gsoc-2024.md:26]: Would be nice to have Slack link here.
```suggestion
Please __do not__ reach out privately to mentors, instead, start a thread in the [`#gsoc-participants`](https://kubeflow.slack.com/archives/C06LW6Z3RA6) channel so others can see the response. Be kind to our mentors, please search to see if your question has already been answered.
```

File: content/en/events/gsoc-2024.md [hunk 12]
```diff
@@ -174,6 +214,8 @@ Training Operator and Katib SDKs have [a valid docstring](https://github.com/kub
 
 **Mentors:** Andrey Velichkevich, Johnu George, Shivay Lamba, Yuan (Terry) Tang, Yuki Iwai
 
+**Component:** Katib/General
```
Comment (andreyvelich) [inline content/en/events/gsoc-2024.md:217]:
```suggestion
**Component:** Katib/Training Operator
```

File: content/en/events/gsoc-2024.md [hunk 13]
```diff
@@ -5,25 +5,53 @@ description = "Google Summer of Code 2024"
...
+  "kubeflow.org_7l5vnbn8suj2se10sen81d9428%40group.calendar.google.com",
+  // KServe Community
+  "4fqdmu5fp4l0bgdlf4lm1atnsl2j4612%40import.calendar.google.com",
+];
+let calender_src = calender_src_list.map(src => `&src=${src}&color=%23A79B8E`).join('');
+const html = `<iframe src="https://calendar.google.com/calendar/embed?ctz=${timezone}&height=600&wkst=1&bgcolor=%23ffffff&showPrint=0&showDate=1&mode=AGENDA&showTitle=0${calender_src}" style="border:solid 1px #777" width="800" height="600" frameborder="0" scrolling="no"></iframe>`;
+document.getElementById('calendar-container').innerHTML = html;
+</script>
+
+## What if my proposal is not chosen?
```
Comment (andreyvelich) [inline content/en/events/gsoc-2024.md:46]: Should we section around contributing guides ?
For example:
Good starting point to learn how to contribute to Kubeflow is to follow [this guide](https://www.kubeflow.org/docs/about/contributing/).
Kubeflow components have individual contributing guide to understand the component better. For example:
- Katib [Contributing Guide](https://github.com/kubeflow/katib/blob/master/docs/developer-guide.md).
- Training Operator [Contributor Guide](https://github.com/kubeflow/training-operator/blob/master/docs/development/developer_guide.md).

Usually, issues with `good first issue` label is beginner friendly tasks that you can start to work on:

- Kubeflow Manifests [good first issues](https://github.com/kubeflow/manifests/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22).
- Training Operator [good first issues](https://github.com/kubeflow/training-operator/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22).
- Katib [good first issues](https://github.com/kubeflow/katib/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22).

Feel free to assign issues to yourself if you want to work on it, leave this comment in the issue `/assign @username`

Comment (andreyvelich) [reply content/en/events/gsoc-2024.md:46]: cc @juliusvonkohout for review

Comment (thesuperzapper) [reply content/en/events/gsoc-2024.md:46]: This might be a little distracting on the page, perhaps we should just add a new section (with this info) to the top of the main contributor guide, and then link it here.

https://www.kubeflow.org/docs/about/contributing/

File: content/en/events/gsoc-2024.md [hunk 14]
```diff
@@ -5,25 +5,53 @@ description = "Google Summer of Code 2024"
...
+Please go to [Google Summer of Code 2024](https://buildyourfuture.withgoogle.com/programs/summer-of-code) and sign up as a student. Next, look at the [projects below](#project-ideas-for-2024-gsoc) to decide which ones you are interested in. Note, that you must submit your proposals through the GSoC website and your proposal must be selected to participate.
 
-## Mentors
+Contributor applications open __March 18th, 2024__ and close on __April 2nd, 2024__.
 
-Kubeflow community is an unparalleled opportunity to make a lasting impact and share your wealth of experience. As a mentor, you'll have the chance to guide and shape the next generation of talent, while also collaborating with like-minded individuals to elevate projects to new heights. With dedicated support mechanisms like community recognition and feedback channels, you'll feel valued and empowered every step of the way. Join us in this exciting journey of mentorship and collaboration! If you want submit an project idea reach out Ramesh Reddy on slack.
+For more information, see the GSoC website and/or reach out to the GSoC organizers.
```
Comment (thesuperzapper) [inline content/en/events/gsoc-2024.md:19]: Let's remove the newlines, to make it a sentence. 

```suggestion
```

File: content/en/events/gsoc-2024.md [hunk 15]
```diff
@@ -5,25 +5,53 @@ description = "Google Summer of Code 2024"
...
+Below are the "ideas/features" from different projects in Kubeflow that are seeking proposals from GSoC Students
 
-## Registration
+## How can I participate in Kubeflow GSoC?
 
-Please go to [Google Summer of Code 2024](https://buildyourfuture.withgoogle.com/programs/summer-of-code) and sign up as student and look up Kubeflow organization and select the projects you want to submit proposals. Note, you can submit more than one proposal. Contributor Applications open **March 20, 2024** and end on **Apr 4th 2024**. So start writing your proposals.
+Please go to [Google Summer of Code 2024](https://buildyourfuture.withgoogle.com/programs/summer-of-code) and sign up as a student. Next, look at the [projects below](#project-ideas-for-2024-gsoc) to decide which ones you are interested in. Note, that you must submit your proposals through the GSoC website and your proposal must be selected to participate.
 
-## Mentors
+Contributor applications open __March 18th, 2024__ and close on __April 2nd, 2024__.
```
Comment (thesuperzapper) [inline content/en/events/gsoc-2024.md:17]: Let's remove the newlines, to make it a sentence. 

```suggestion
```

File: content/en/events/gsoc-2024.md [hunk 16]
```diff
@@ -5,25 +5,53 @@ description = "Google Summer of Code 2024"
 
 ---
 
-Kubeflow Community is excited to announce that we are applying this year to participate in the [**Google Summer of Code 2024**](https://buildyourfuture.withgoogle.com/programs/summer-of-code)
+Kubeflow Community is excited to announce that we have been [selected](https://summerofcode.withgoogle.com/programs/2024/organizations/kubeflow) as organization to participate in [**Google Summer of Code 2024**](https://buildyourfuture.withgoogle.com/programs/summer-of-code).
```
Comment (thesuperzapper) [inline content/en/events/gsoc-2024.md:9]: Let's remove the newlines, to make it a sentence. 

```suggestion
```

File: content/en/events/gsoc-2024.md [hunk 17]
```diff
@@ -5,25 +5,53 @@ description = "Google Summer of Code 2024"
...
 
-Below are the "ideas/features" from different projects in the Kubeflow that being submitted for seeking proposals from GSoC Students.
+Below are the "ideas/features" from different projects in Kubeflow that are seeking proposals from GSoC Students
```
Comment (thesuperzapper) [inline content/en/events/gsoc-2024.md:10]: Its probably not necessary to say this, and putting things in quotes looks a bit strange.

```suggestion
This page aims to help you participate in the Kubeflow organization for GSoC 2024.
```

File: content/en/events/gsoc-2024.md [hunk 18]
```diff
@@ -5,25 +5,53 @@ description = "Google Summer of Code 2024"
...
+document.getElementById('calendar-container').innerHTML = html;
+</script>
+
+## What if my proposal is not chosen?
+
+Please understand that not everyone can be selected for Google Summer of Code (GSoC), there are many possible candidates for each project.
+
+However, we want to encourage you to participate in the Kubeflow project!
+
+The best way to get started is to attend [working group meetings](/docs/about/community/#kubeflow-community-calendars) for components you want to contribute to.
```
Comment (thesuperzapper) [inline content/en/events/gsoc-2024.md:52]: Removed the newline, and slightly reworded:

```suggestion
However, we still want to encourage you to participate in the Kubeflow project!
Get started by attending [working group meetings](/docs/about/community/#kubeflow-community-calendars) for components you want to help with, and reading our [contributing guide](/docs/about/contributing/).
```

File: content/en/events/gsoc-2024.md (hunk 11 above)
Comment (thesuperzapper) [reply content/en/events/gsoc-2024.md:26]: agree

**Review by thesuperzapper — CHANGES_REQUESTED** [review]
@rareddy found a few more things, and we should be good to merge after that.

File: content/en/events/gsoc-2024.md (hunk 11 above)
Comment (rareddy) [reply content/en/events/gsoc-2024.md]: updated now.
